packages/babel-helper-function-name/src/index.ts

```diff
@@ -2,7 +2,7 @@ import getFunctionArity from "@babel/helper-get-function-arity";
 import template from "@babel/template";
 import * as t from "@babel/types";
 
-const buildPropertyMethodAssignmentWrapper = template(`
+const buildPropertyMethodAssignmentWrapper = template.statement(`
```

Comment:
Q: Why do we need this change?

Reply:
this way it was more granular type (just `t.Statement` instead of union of all possible options - I remember in some place ts was complaining about possible array instead of ast node), however - later I anyway added more typecasts… so this is no longer needed - reverting it